open_academy/views/open_academy_session_views.xml

```diff
@@ -36,16 +40,86 @@
                             <group colspan="10" col="4">
                                 <field name="description" />
                                 <newline /> 
+                                <field name="active"/>
+                                <newline /> 
                                 <field name="start_date"/>
                                 <newline /> 
+                                <field name="end_date"/>
+                                <newline />
                                 <field name="duration"/>
                                 <newline /> 
                                 <field name="number_of_seats"/>
+                                <newline /> 
+                                <field name="instructor_id" domain="[ '|', ('instructor', '=' , True), '&amp;', ('category_id.name', 'ilike', 'level%'), ('category_id.parent_id.name', 'ilike', 'teacher')]"/>
+                                <newline /> 
+                                <field name="course_id"/>
+                                <newline /> 
+                                <field name="attendees_ids"/>
+                                <newline /> 
+                                <field name="percentage_of_seats_taken" widget="progressbar"/>
+                                <newline /> 
                             </group>
                         </page>
                     </notebook>
                 </sheet>
             </form>
         </field>
     </record>
+    <!-- calendar configuration -->
+    <record model="ir.ui.view" id="open_academy_session_view_calendar">
+        <field name="name">openacademy.session.view_calendar</field>
+        <field name="model">openacademy.session</field>
+        <field name="arch" type="xml">
+            <calendar string="Sessions" date_start="start_date"  date_stop="end_date"  color="course_id">
```

Comment:
![image](https://user-images.githubusercontent.com/113544718/194970909-390a6137-f587-464a-bfda-949916250d73.png)

"Session name" appears in the header of the pop-up message. I did not put it because it seemed redundant to me.

Reply:
Excellent. I did not know that the name already appears, by default, in previous versions this did not happen.